emails/index.tsx

```diff
@@ -0,0 +1,200 @@
+import { EventRequestInfo } from "@/types/eventRequestInfo";
+import {
+    Body,
+    Container,
+    Column,
+    Head,
+    Html,
+    Preview,
+    Row,
+    Section,
+    Text,
+    Hr,
+  } from "@react-email/components";
+  import * as React from "react";
+  
+  export const AppleReceiptEmail = ({requestEventInfo} : {requestEventInfo: EventRequestInfo }) => (
+    <Html>
+      <Head />
+      <Preview>test</Preview>
+  
+      <Body style={main}>
+        <Container style={container}>
+          <Section>
+            <Row>
+              <Column style={tableCell}>
+                <Text style={heading}> {requestEventInfo.isEvent ? "Marcação de Evento" : "Pedido de Material"}</Text>
+              </Column>
+            </Row>
+          </Section>
```

Comment:
Why not using tailwind?

Reply:
Styling for this page was already present in the template.